lib/alephant/logger/level.rb

```diff
@@ -0,0 +1,35 @@
+module Alephant
+  module Logger
+    LEVELS = %i(debug info warn error).freeze
```

Comment:
If this was a supervisor/controller, then it would make more sense to contain the levels here:
```
Alephant::Logger::LevelSupervisor::Levels
```

It seemed a bit of an odd place otherwise, as these seemed like they would belong inside Alephant::Logger directly... I think renaming this class and making it static (as previous comment) would allow these levels to be defined inside and be a part of the class.

Reply:
Aaah, good call. Makes sense to have it in the static class. 👍